{"code":101,"level":"error","message":"Moralis auth failed, invalid data","stack":"Error: Moralis auth failed, invalid data\n    at I:\\000_WWW\\dinocapitaldatabase\\src\\auth\\MoralisEthAdapter.ts:32:13\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async Promise.all (index 0)","timestamp":"2023-01-29T14:24:02.153Z"}
{"code":101,"level":"error","message":"Moralis auth failed, invalid data","stack":"Error: Moralis auth failed, invalid data\n    at I:\\000_WWW\\dinocapitaldatabase\\src\\auth\\MoralisEthAdapter.ts:32:13\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async Promise.all (index 0)","timestamp":"2023-01-29T14:36:24.473Z"}
{"code":101,"level":"error","message":"Moralis auth failed, invalid data","stack":"Error: Moralis auth failed, invalid data\n    at I:\\000_WWW\\dinocapitaldatabase\\src\\auth\\MoralisEthAdapter.ts:32:13\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async Promise.all (index 0)","timestamp":"2023-01-29T15:21:41.234Z"}
{"code":101,"level":"error","message":"Moralis auth failed, invalid data","stack":"Error: Moralis auth failed, invalid data\n    at I:\\000_WWW\\dinocapitaldatabase\\src\\auth\\MoralisEthAdapter.ts:32:13\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async Promise.all (index 0)","timestamp":"2023-01-29T15:38:47.622Z"}
{"code":101,"level":"error","message":"Moralis auth failed, invalid data","stack":"Error: Moralis auth failed, invalid data\n    at I:\\000_WWW\\dinocapitaldatabase\\src\\auth\\MoralisEthAdapter.ts:32:13\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async Promise.all (index 0)","timestamp":"2023-01-29T15:40:44.324Z"}
{"code":101,"level":"error","message":"Moralis auth failed, invalid data","stack":"Error: Moralis auth failed, invalid data\n    at I:\\000_WWW\\dinocapitaldatabase\\src\\auth\\MoralisEthAdapter.ts:32:13\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async Promise.all (index 0)","timestamp":"2023-01-29T18:16:37.127Z"}
{"code":101,"level":"error","message":"Moralis auth failed, invalid data","stack":"Error: Moralis auth failed, invalid data\n    at I:\\000_WWW\\dinocapitaldatabase\\src\\auth\\MoralisEthAdapter.ts:32:13\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async Promise.all (index 0)","timestamp":"2023-01-29T18:22:54.764Z"}
{"code":101,"level":"error","message":"Moralis auth failed, invalid data","stack":"Error: Moralis auth failed, invalid data\n    at I:\\000_WWW\\dinocapitaldatabase\\src\\auth\\MoralisEthAdapter.ts:32:13\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async Promise.all (index 0)","timestamp":"2023-01-29T18:36:41.397Z"}
{"code":101,"level":"error","message":"Moralis auth failed, invalid data","stack":"Error: Moralis auth failed, invalid data\n    at I:\\000_WWW\\dinocapitaldatabase\\src\\auth\\MoralisEthAdapter.ts:32:13\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async Promise.all (index 0)","timestamp":"2023-01-29T18:37:22.003Z"}
{"code":101,"level":"error","message":"Moralis auth failed, invalid data","stack":"Error: Moralis auth failed, invalid data\n    at I:\\000_WWW\\dinocapitaldatabase\\src\\auth\\MoralisEthAdapter.ts:32:13\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async Promise.all (index 0)","timestamp":"2023-01-29T18:41:13.472Z"}
{"code":101,"level":"error","message":"Moralis auth failed, invalid data","stack":"Error: Moralis auth failed, invalid data\n    at I:\\000_WWW\\dinocapitaldatabase\\src\\auth\\MoralisEthAdapter.ts:32:13\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async Promise.all (index 0)","timestamp":"2023-01-29T18:45:58.686Z"}
{"code":101,"level":"error","message":"Moralis auth failed, invalid data","stack":"Error: Moralis auth failed, invalid data\n    at I:\\000_WWW\\dinocapitaldatabase\\src\\auth\\MoralisEthAdapter.ts:32:13\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async Promise.all (index 0)","timestamp":"2023-01-29T18:50:23.051Z"}
{"code":101,"level":"error","message":"Moralis auth failed, invalid data","stack":"Error: Moralis auth failed, invalid data\n    at I:\\000_WWW\\dinocapitaldatabase\\src\\auth\\MoralisEthAdapter.ts:32:13\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async Promise.all (index 0)","timestamp":"2023-01-29T19:04:11.481Z"}
{"error":{"code":141,"message":"[C0006] Request failed: getaddrinfo ENOTFOUND authapi.moralis.io"},"functionName":"requestMessage","level":"error","message":"Failed running cloud function requestMessage for user undefined with:\n  Input: {\"address\":\"0xbfc21f7272e36f3973aa5219c0d95beb3a7ef0d9\",\"chain\":137,\"network\":\"evm\"}\n  Error: {\"message\":\"[C0006] Request failed: getaddrinfo ENOTFOUND authapi.moralis.io\",\"code\":141}","params":{"address":"0xbfc21f7272e36f3973aa5219c0d95beb3a7ef0d9","chain":137,"network":"evm"},"timestamp":"2023-01-29T19:39:55.104Z"}
{"code":141,"level":"error","message":"[C0006] Request failed: getaddrinfo ENOTFOUND authapi.moralis.io","stack":"Moralis SDK Core Error: [C0006] Request failed: getaddrinfo ENOTFOUND authapi.moralis.io\n    at RequestController.makeError (I:\\000_WWW\\dinocapitaldatabase\\node_modules\\@moralisweb3\\common-core\\src\\controllers\\RequestController\\RequestController.ts:88:12)\n    at RequestController.<anonymous> (I:\\000_WWW\\dinocapitaldatabase\\node_modules\\@moralisweb3\\common-core\\src\\controllers\\RequestController\\RequestController.ts:56:26)\n    at step (I:\\000_WWW\\dinocapitaldatabase\\node_modules\\@moralisweb3\\common-core\\lib\\controllers\\RequestController\\RequestController.js:44:23)\n    at Object.throw (I:\\000_WWW\\dinocapitaldatabase\\node_modules\\@moralisweb3\\common-core\\lib\\controllers\\RequestController\\RequestController.js:25:53)\n    at rejected (I:\\000_WWW\\dinocapitaldatabase\\node_modules\\@moralisweb3\\common-core\\lib\\controllers\\RequestController\\RequestController.js:17:65)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)","timestamp":"2023-01-29T19:39:55.115Z"}
{"error":{"code":141,"message":"[C0006] Request failed: getaddrinfo ENOTFOUND authapi.moralis.io"},"functionName":"requestMessage","level":"error","message":"Failed running cloud function requestMessage for user undefined with:\n  Input: {\"address\":\"0xbfc21f7272e36f3973aa5219c0d95beb3a7ef0d9\",\"chain\":137,\"network\":\"evm\"}\n  Error: {\"message\":\"[C0006] Request failed: getaddrinfo ENOTFOUND authapi.moralis.io\",\"code\":141}","params":{"address":"0xbfc21f7272e36f3973aa5219c0d95beb3a7ef0d9","chain":137,"network":"evm"},"timestamp":"2023-01-29T19:39:55.209Z"}
{"code":141,"level":"error","message":"[C0006] Request failed: getaddrinfo ENOTFOUND authapi.moralis.io","stack":"Moralis SDK Core Error: [C0006] Request failed: getaddrinfo ENOTFOUND authapi.moralis.io\n    at RequestController.makeError (I:\\000_WWW\\dinocapitaldatabase\\node_modules\\@moralisweb3\\common-core\\src\\controllers\\RequestController\\RequestController.ts:88:12)\n    at RequestController.<anonymous> (I:\\000_WWW\\dinocapitaldatabase\\node_modules\\@moralisweb3\\common-core\\src\\controllers\\RequestController\\RequestController.ts:56:26)\n    at step (I:\\000_WWW\\dinocapitaldatabase\\node_modules\\@moralisweb3\\common-core\\lib\\controllers\\RequestController\\RequestController.js:44:23)\n    at Object.throw (I:\\000_WWW\\dinocapitaldatabase\\node_modules\\@moralisweb3\\common-core\\lib\\controllers\\RequestController\\RequestController.js:25:53)\n    at rejected (I:\\000_WWW\\dinocapitaldatabase\\node_modules\\@moralisweb3\\common-core\\lib\\controllers\\RequestController\\RequestController.js:17:65)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)","timestamp":"2023-01-29T19:39:55.252Z"}
{"error":{"code":141,"message":"[C0006] Request failed: getaddrinfo ENOTFOUND authapi.moralis.io"},"functionName":"requestMessage","level":"error","message":"Failed running cloud function requestMessage for user undefined with:\n  Input: {\"address\":\"0xbfc21f7272e36f3973aa5219c0d95beb3a7ef0d9\",\"chain\":137,\"network\":\"evm\"}\n  Error: {\"message\":\"[C0006] Request failed: getaddrinfo ENOTFOUND authapi.moralis.io\",\"code\":141}","params":{"address":"0xbfc21f7272e36f3973aa5219c0d95beb3a7ef0d9","chain":137,"network":"evm"},"timestamp":"2023-01-29T19:39:55.303Z"}
{"code":141,"level":"error","message":"[C0006] Request failed: getaddrinfo ENOTFOUND authapi.moralis.io","stack":"Moralis SDK Core Error: [C0006] Request failed: getaddrinfo ENOTFOUND authapi.moralis.io\n    at RequestController.makeError (I:\\000_WWW\\dinocapitaldatabase\\node_modules\\@moralisweb3\\common-core\\src\\controllers\\RequestController\\RequestController.ts:88:12)\n    at RequestController.<anonymous> (I:\\000_WWW\\dinocapitaldatabase\\node_modules\\@moralisweb3\\common-core\\src\\controllers\\RequestController\\RequestController.ts:56:26)\n    at step (I:\\000_WWW\\dinocapitaldatabase\\node_modules\\@moralisweb3\\common-core\\lib\\controllers\\RequestController\\RequestController.js:44:23)\n    at Object.throw (I:\\000_WWW\\dinocapitaldatabase\\node_modules\\@moralisweb3\\common-core\\lib\\controllers\\RequestController\\RequestController.js:25:53)\n    at rejected (I:\\000_WWW\\dinocapitaldatabase\\node_modules\\@moralisweb3\\common-core\\lib\\controllers\\RequestController\\RequestController.js:17:65)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)","timestamp":"2023-01-29T19:39:55.338Z"}
{"error":{"code":141,"message":"[C0006] Request failed: timeout of 20000ms exceeded"},"functionName":"requestMessage","level":"error","message":"Failed running cloud function requestMessage for user undefined with:\n  Input: {\"address\":\"0xbfc21f7272e36f3973aa5219c0d95beb3a7ef0d9\",\"chain\":137,\"network\":\"evm\"}\n  Error: {\"message\":\"[C0006] Request failed: timeout of 20000ms exceeded\",\"code\":141}","params":{"address":"0xbfc21f7272e36f3973aa5219c0d95beb3a7ef0d9","chain":137,"network":"evm"},"timestamp":"2023-01-29T19:39:56.058Z"}
{"code":141,"level":"error","message":"[C0006] Request failed: timeout of 20000ms exceeded","stack":"Moralis SDK Core Error: [C0006] Request failed: timeout of 20000ms exceeded\n    at RequestController.makeError (I:\\000_WWW\\dinocapitaldatabase\\node_modules\\@moralisweb3\\common-core\\src\\controllers\\RequestController\\RequestController.ts:88:12)\n    at RequestController.<anonymous> (I:\\000_WWW\\dinocapitaldatabase\\node_modules\\@moralisweb3\\common-core\\src\\controllers\\RequestController\\RequestController.ts:56:26)\n    at step (I:\\000_WWW\\dinocapitaldatabase\\node_modules\\@moralisweb3\\common-core\\lib\\controllers\\RequestController\\RequestController.js:44:23)\n    at Object.throw (I:\\000_WWW\\dinocapitaldatabase\\node_modules\\@moralisweb3\\common-core\\lib\\controllers\\RequestController\\RequestController.js:25:53)\n    at rejected (I:\\000_WWW\\dinocapitaldatabase\\node_modules\\@moralisweb3\\common-core\\lib\\controllers\\RequestController\\RequestController.js:17:65)\n    at runNextTicks (node:internal/process/task_queues:61:5)\n    at listOnTimeout (node:internal/timers:528:9)\n    at processTimers (node:internal/timers:502:7)","timestamp":"2023-01-29T19:39:56.087Z"}
{"error":{"code":141,"message":"[C0006] Request failed: getaddrinfo ENOTFOUND authapi.moralis.io"},"functionName":"requestMessage","level":"error","message":"Failed running cloud function requestMessage for user undefined with:\n  Input: {\"address\":\"0xbfc21f7272e36f3973aa5219c0d95beb3a7ef0d9\",\"chain\":137,\"network\":\"evm\"}\n  Error: {\"message\":\"[C0006] Request failed: getaddrinfo ENOTFOUND authapi.moralis.io\",\"code\":141}","params":{"address":"0xbfc21f7272e36f3973aa5219c0d95beb3a7ef0d9","chain":137,"network":"evm"},"timestamp":"2023-01-29T19:39:59.443Z"}
{"code":141,"level":"error","message":"[C0006] Request failed: getaddrinfo ENOTFOUND authapi.moralis.io","stack":"Moralis SDK Core Error: [C0006] Request failed: getaddrinfo ENOTFOUND authapi.moralis.io\n    at RequestController.makeError (I:\\000_WWW\\dinocapitaldatabase\\node_modules\\@moralisweb3\\common-core\\src\\controllers\\RequestController\\RequestController.ts:88:12)\n    at RequestController.<anonymous> (I:\\000_WWW\\dinocapitaldatabase\\node_modules\\@moralisweb3\\common-core\\src\\controllers\\RequestController\\RequestController.ts:56:26)\n    at step (I:\\000_WWW\\dinocapitaldatabase\\node_modules\\@moralisweb3\\common-core\\lib\\controllers\\RequestController\\RequestController.js:44:23)\n    at Object.throw (I:\\000_WWW\\dinocapitaldatabase\\node_modules\\@moralisweb3\\common-core\\lib\\controllers\\RequestController\\RequestController.js:25:53)\n    at rejected (I:\\000_WWW\\dinocapitaldatabase\\node_modules\\@moralisweb3\\common-core\\lib\\controllers\\RequestController\\RequestController.js:17:65)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)","timestamp":"2023-01-29T19:39:59.448Z"}
